lms/djangoapps/instructor_task/views.py

```diff
@@ -219,14 +221,32 @@ def get_task_completion_info(instructor_task):  # lint-amnesty, pylint: disable=
     if student is None and num_attempted != num_total:
         # Translators: {total} is a count.  This message is appended to task progress status messages.
         msg_format += _(" (out of {total})")
+    
+    detailed_failure_msg = ""
+    detailed_skip_msg = ""
+
+    if failure_info:
+        detailed_failure_msg = "Detailed Failures: " + ", ".join([f"{email}: {reason}" for email, reason in failure_info])
+
+    if skip_info:
+        detailed_skip_msg = "Detailed Skips: " + ", ".join([f"{reason}: {email}" for email, reason in skip_info])
+
+    if detailed_failure_msg and detailed_skip_msg:
+        detailed_msg = detailed_failure_msg + "; " + detailed_skip_msg
+    elif detailed_failure_msg:
+        detailed_msg = detailed_failure_msg
+    elif detailed_skip_msg:
+        detailed_msg = detailed_skip_msg
+    else:
+        detailed_msg = "No detailed failure or skip information available."
```

Comment:
nit: this is a redundant `else` statement. We can initialize `detailed_msg` with this value.